app/_src/gateway/production/networking/troubleshoot-tls.md

```diff
@@ -0,0 +1,146 @@
+FATAL:  certificate authentication failed for user "kong"
+```
+
+This error message happens when the client certificate doesn't match the username specified in the database. 
+
+### Connection requires a valid client certificate
+
+```sh
+FATAL:  connection requires a valid client certificate
+```
+
+This error occurs when the PostgreSQL server requires a client certificate for authentication, but the client, in this case {{site.base_gateway}}, was unable to provide a valid certificate. For this type of error, check if `pg_ssl_cert` and `pg_ssl_cert_key` are correctly set in `kong.conf`. 
+
+
+
+## Troubleshooting TLS on {{site.base_gateway}}
+
+### Failed to retrieve PostgreSQL version 
+
+If you receive either of the following error messages: 
+
+```sh
+Error: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: tlsv1 alert protocol version
+```
+
+```sh
+Error: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: unsupported protocol
+```
+The server and client versions are not in sync. Check the version that the PostgreSQL server is using and adjust the {{site.base_gateway}} version accordingly. 
+
+
+
+
+### The server does not support SSL connections
+
+```sh
+Error: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: the server does not support SSL connections
+```
+
+This error occurs when SSL is not supported by PostgreSQL. Configure PostgreSQL according to [the configuration instructions](/gateway/{{page.kong_version}}/production/networking/configure-tls/).
+
+
+
+### Certificate verify failed 
+
+```sh
+Error: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: certificate verify failed
+```
+This error occurs when {{site.base_gateway}} fails to verify the PostgreSQL server certificate. Ensure that the PostgreSQL server has configured a trusted certificate and the corresponding CA chain is correctly set in `lua_ssl_trusted_certificate` within `kong.conf`. 
+
+
+### Connection requires a valid client certificate
+
+```sh
+Error: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: FATAL: connection requires a valid client certificate
+```
+{{site.base_gateway}} requires a client certificate for authentication, but the client was unable to provide a valid certificate. 
+
+This error occurs when the {{site.base_gateway}} server requires a client certificate for authentication, but the server is unable to provide a valid certificate. For this type of error, check if `pg_ssl_cert` and `pg_ssl_cert_key` are correctly set in `kong.conf`. 
+
+
+### Failed to retrieve PostgreSQL version
+
+```sh
+Error: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: tlsv1 alert unknown ca
+```
+The PostgreSQL server failed to verify the client certificate. Use a client certificate which is trusted by the CA set in `ssl_ca_file` within `postgres.conf`. 
+
+### Certificate authentication failed for user
+
```

Comment:
```suggestion
```sh
Error: [PostgreSQL error] failed to retrieve PostgreSQL server_version_num: FATAL: certificate authentication failed for user "kong"
```

```